src/option.h

```diff
@@ -83,7 +83,7 @@ namespace librealsense
         std::string _desc;
     };
 
-    class LRS_EXTENSION_API option_base : public option
+    class LRS_EXTENSION_API option_base : public virtual option
```

Comment:
This implies multiple option inheritance that shouldn't addressed and avoided by design.